templates/server/config.ru.erb

```diff
@@ -18,6 +18,9 @@ ARGV << "--rack"
 # to ~puppet/.puppet
 ARGV << "--confdir" << "<%= scope.lookupvar('::puppet::server_dir') %>"
 ARGV << "--vardir"  << "<%= scope.lookupvar('::puppet::server_vardir') %>"
+<% (scope.lookupvar('::puppet::server_rack_arguments') || []).each do |server_rack_argument| -%>
```

Comment:
I think this is causing the test failure, as server_rack_arguments can be `:undefined` (or `:undef`)